src/game/scripts/vscripts/abilities/bash_cooldown.lua

```diff
@@ -0,0 +1,28 @@
+LinkLuaModifier('modifier_bash_cooldown', 'abilities/bash_cooldown.lua', LUA_MODIFIER_MOTION_NONE)
+
+modifier_bash_cooldown = class({
+	IsPurgable = function() return false end,
+	GetTexture = function() return 'spirit_breaker_greater_bash' end,
+})
+
+function BashCooldown( filterTable )
+    local parent = EntIndexToHScript(filterTable.entindex_parent_const)
+    local caster = EntIndexToHScript(filterTable.entindex_caster_const)
+    local ability = EntIndexToHScript(filterTable.entindex_ability_const)
```

Comment:
if there is no `entindex_ability_const ` this will cause an error, same with parent and caster index

Reply:
Can you upload a fix to the main branch?